config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "users#posts"
```

Comment:
:red_circle: [REQUIRED] [As per this requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md). When you run `rails routes `command in your terminal, you should see the following output:
![routes](https://github.com/saluumaa/blog-app/assets/26736582/8974eaf4-8300-42be-81a7-3f170394c254)
After running that command I can't find the first route 
` user_posts GET    /users/:user_id/posts(.:format)    posts#index`.
To fix this, you need to update your config/routes.rb file. Please make sure the` resources :users` method does  define a nested resource called user_posts as follow: :arrow_down: 

```
resources :users, only: [] do
  resources :posts, only: [:show, :index]
end
```


Please fix this missing route :hugs: